Astik87/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,27 @@
+import './Header.scss';
+
+import React, { Component } from 'react';
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Typography from '@material-ui/core/Typography';
+import { Link } from 'react-router-dom';
+
+export class Header extends Component {
+
+    render() {
+        return (
+            <div className="header">
+              <AppBar position="static">
+                <Toolbar>
+                  <Typography variant="h6" >
+                    <Link className='menu-item' to='/' >Chats</Link>
```

Comment:
В аттрибутах лучше использовать двойные кавычки
```suggestion
                    <Link className="menu-item" to="/" >Chats</Link>
```